### PR TITLE
Twelve-Days: Punctuation fix

### LINKED ISSUE #1328: Twelve-Days: Puncutation error
In the Twelve Days of Christmas, there should be a colon, not a comma, after 'gave to me.' A colon more accurately reflects the rhythm of the song as well as grammatically indicates that the first part of the sentence is meant to set up the second part.

### PR DESCRIPTION
**Pull request by PatrickMcSweeny**
Closes #1328

### CODE REVIEW

**Comment by PatrickMcSweeny** [comment]
@sdublish Done

File: exercises/twelve-days/canonical-data.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
             "endVerse": 2
           },
           "expected": [
-            "On the second day of Christmas my true love gave to me, two Turtle Doves, and a Partridge in a Pear Tree."
+            "On the second day of Christmas my true love gave to me: two Turtle Doves, and a Partridge in a Pear Tree."
```
Comment (rpottsoh) [inline exercises/twelve-days/canonical-data.json]: `two`, `three`, `four` etc. should all also be capitalized if want to be really picky.
https://en.wikipedia.org/wiki/The_Twelve_Days_of_Christmas_(song)#/media/File:12_days_melody.png

Comment (PatrickMcSweeny) [reply exercises/twelve-days/canonical-data.json]: Personally I think it'd be better to lowercase all of the items.

**Comment by rpottsoh** [comment]
@PatrickMcSweeny I am going to go ahead and merge your PR.  Thanks for taking the time to make this exercise a little bit better.  :+1:

**Pull request merged**